### PR TITLE
Add drag auto scroll capability for custom scrollbar containers

### PR DESCRIPTION
**Pull request by nicoback2**
### Description
autoscroll

Sadly the new scrollbar component doesn't play well with native drag and drop scrolling - see https://github.com/mdbootstrap/perfect-scrollbar/issues/100 and https://stackoverflow.com/questions/55482184/auto-scroll-while-dragging-an-item-with-perfect-scrollbar-and-sortablejs.

This means that when you drag something to the top or bottom of the nav container, it doesn't scroll automatically like it used to.
This PR adds back that functionality manually (+ a shadow indicator when you're drag-scrolling up or down).

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?
Tested in browser
*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### CODE REVIEW

File: packages/web/src/components/drag-autoscroller/DragAutoscroller.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
+import React, { useRef, useState } from 'react'
```
Comment (nicoback2) [inline packages/web/src/components/drag-autoscroller/DragAutoscroller.tsx:1]: Wonder if this should go in Stems instead? 🤔

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--autoscoll-dnd

File: packages/web/src/components/drag-autoscroller/DragAutoscroller.tsx (hunk 1 above)
Comment (rickyrombo) [reply packages/web/src/components/drag-autoscroller/DragAutoscroller.tsx:1]: I think that makes sense. We can start here though and graduate it if we see it needed elsewhere

**Review by rickyrombo — APPROVED** [review]
Some nits for the "magic numbers" but otherwise looks good!

**Comment by raymondjacobson** [comment]
pls bug julian for the UI if you haven't already!

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/nkang--autoscoll-dnd

**Pull request merged**